### DISCUSSION TITLE
Laravel mix

### DISCUSSION THREAD

**Discussion by Rapiddox** [tailwindlabs/tailwindcss#2817] [Help] [answered]
Hello guys.
Somebody can help me in that how can i use laravel mix with sass and tailwindcss 2?
Here a screenshot how my setup was with tailwind v1.* and it worked before, but with tailwindcss v2.* it doesn't work.

![image](https://user-images.githubusercontent.com/25752233/99675959-29c2b380-2a78-11eb-8e82-8adfb94c7173.png)

**Comment by Rapiddox** [comment, marked as answer]
It will work when laravel-mix 6 will be released. (works with "laravel-mix": "^6.0.0-beta.14")

**Comment by vincentcordel** [comment]
what's not working? I have the same setup without problem.

**Comment by adamwathan** [comment]
Laravel Mix v5 requires the compatibility build:

https://tailwindcss.com/docs/installation#post-css-7-compatibility-build
